Cython/Compiler/ExprNodes.py

```diff
@@ -9839,6 +9839,10 @@ def generate_result_code(self, code):
             flags.append('CO_VARARGS')
         if self.def_node.starstar_arg:
             flags.append('CO_VARKEYWORDS')
+        if self.def_node.is_generator and self.def_node.is_coroutine:
+            flags.append('CO_COROUTINE')
+            if self.def_node.is_asyncgen:
+                flags.append('CO_ASYNC_GENERATOR')
```

Comment:
I think it should be this:

```suggestion
        if self.def_node.is_asyncgen:
            flags.append('CO_ASYNC_GENERATOR')
        elif self.def_node.is_coroutine:
            flags.append('CO_COROUTINE')
        elif self.def_node.is_generator:
            flags.append('CO_GENERATOR')
```

Reply:
Ok, that seems better then. Thanks.